sqlalchemy_ibmi/base.py

```diff
@@ -258,40 +258,16 @@ class XML(sa_types.Text):
 class DB2TypeCompiler(compiler.GenericTypeCompiler):
     """IBM i Db2 Type Compiler"""
 
-    def visit_TIMESTAMP(self, type_, **kw):
-        return "TIMESTAMP"
-
-    def visit_DATE(self, type_, **kw):
-        return "DATE"
-
-    def visit_TIME(self, type_, **kw):
```

Comment:
Does the upstream TIME type support precision? If so, we should keep this and raise an exception if the user specifies a precision.

Reply:
It looks like it does not